### DISCUSSION TITLE
Propshaft for Qute

### DISCUSSION THREAD

**Discussion by akil-rails** [quarkusio/quarkus#45183] [Q&A]
Rails 8 introduced https://github.com/rails/propshaft for adding digests to vanilla js & css files. Will be great to have that in Qute as well.

[a comment by an automated account is not shown]

**Comment by mkouba** [comment]
CC @ia3andy

**Comment by ia3andy** [comment]
That's pretty much what the web-bundler does AFAIK https://docs.quarkiverse.io/quarkus-web-bundler/dev/advanced-guides.html#bundle-paths?
